Assets/MRTK/SDK/Features/UX/Scripts/Pointers/PokePointer.cs

```diff
@@ -452,6 +458,17 @@ public override void OnSourceDetected(SourceStateEventData eventData)
             }
         }
 
+        /// <inheritdoc />
+        public override void OnSourcePoseChanged(SourcePoseEventData<MixedRealityPose> eventData)
+        {
+            base.OnSourcePoseChanged(eventData);
+
+            if (SourcePoseDataUsable(eventData))
```

Comment:
super nit, but it seems unfortunate that both the `base` call and this call into `SourcePoseDataUsable`. Not sure a better pattern offhand though, since the base doesn't return anything...

Reply:
though kind of annoying pattern wise I think it should be good enough for the current iteration